docs/contents/components/other/close-button/accessibility.ja.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: CloseButton
+description: "CloseButtonは、基本的にコンポーネントの閉じる機能をトリガーするために使用するコンポーネントです。"
+package-name: "@yamada-ui/close-button"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`CloseButton`は、アクセシビリティに関して[WAI-ARIA Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/)に従います。
```

Comment:
![nits](https://img.shields.io/badge/review-nits-inactive.svg)

I noticed that the link text is slightly different in the Japanese version and the English version.

I think it is more natural to use the same text in both languages:

```suggestion
`CloseButton`は、アクセシビリティに関して[WAI-ARIA - Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/)に従います。
```

#### Reference

Other components follow the `WAI-ARIA - XXX Pattern` style:

https://github.com/yamada-ui/yamada-ui/blob/44f096087fd3917d479f36f6c5475a270006b33c/docs/contents/components/disclosure/accordion/accessibility.ja.mdx?plain=1#L10

https://github.com/yamada-ui/yamada-ui/blob/44f096087fd3917d479f36f6c5475a270006b33c/docs/contents/components/disclosure/tabs/accessibility.ja.mdx?plain=1#L10

Reply:
I fixed it.
Thank you.